Readme.md

```diff
@@ -1,21 +1,146 @@
-# Aragon Core
+# Aragon OSx Protool
 
-This workspace contains the Core foundation of the Aragon ecosystem - the human centered approach to DAO's.
+The Aragon OSx protocol is the foundation layer of the new Aragon stack. It allows users to create, manage, and customize DAOs in a way that is lean, adaptable, and secure.
 
-For more information on the individual packages, please read the respective `Readme.md`.
+Within this monorepo, you will be able to find 3 individual packages:
+- [Contracts](https://github.com/aragon/osx/tree/develop/packages/contracts): the Aragon OSx protocol contracts.
+- [Subgraph](https://github.com/aragon/osx/tree/develop/packages/subgraph): containing all code generating our subgraph and event indexing.
+- [Contract-ethers](https://github.com/aragon/osx/tree/develop/packages/contracts-ethers): containing the connection between the ethers package and our contracts.
+
+For more information on the individual packages, please read the respective `README.md`.
+
+## Contributing
+
+We'd love to hear what you think! If you want to build this with us, please find a detailed contribution guide in the `CONTRIBUTION_GUIDE.md` [file here](https://github.com/aragon/osx/blob/develop/CONTRIBUTION_GUIDE.md).
 
 ## Setup
 
-Start by running `yarn install` in the root.
+Start by running `yarn install` in the root in your terminal.
 
 ### Dependencies
 
 Since the repo is set up as yarn workspace, all the linking is done automatically.
 
+## How the Aragon OSx protocol works
+
+To review the contracts powering the Aragon OSx protocol, feel free to head to `packages/contracts`.
+
+The Aragon OSx protocol architecture is composed of two key sections:
+
+- __Core contracts__: details the functionalities all DAOs will have. It is composed of mostly 3 sections:
+    - **DAO treasury:** the DAO’s main contract and vault.
+    - **Permission manager**: manages permissions within the DAO to execute actions.
+    - **Plugin manager**: manages installed plugins within the DAO.
```

Comment:
What do you mean by `Plugin manager`?
The folder `packages/contracts/src/core/plugin` contains various helpers and base contracts for plugin development

Reply:
@Michael-A-Heuer FYI

Reply:
I also find this a bit confusing.

The Aragon OSx protocol consists of the core and framework contracts.
The core contracts are the primitives the end-user interacts with
- The DAO contract
- Plugin templates to build upon
- Permissions to govern the interactions between the plugins, DAOs, but also other 

The Aragon OSx framework contracts that run the protocol and streamline the processes of
- DAO creation and 
- plugin management